packages/composite-checkout-wpcom/src/components/form-field-annotation.tsx

```diff
@@ -0,0 +1,117 @@
+/**
+ * External dependencies
+ */
+import React, { FunctionComponent } from 'react';
+import styled from '@emotion/styled';
+
+type FormFieldAnnotationProps = {
+	// Semantic props
+	labelText: string;
+	descriptionText?: string;
+	errorMessage: string;
+
+	// Functional props
+	isError: boolean;
+	isDisabled: boolean;
```

Comment:
Will it pass type checking if I omit the `isError` and `isDisabled` props and let them have default values? Usually, props like this are optional and have good defaults.

Reply:
I made these optional, although in practice this will be a controlled component so the parent probably should be passing in values for `isError` and `isDisabled`.